react/ProductQuantity.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { useItemContext } from './components/ItemContext'
+
+const ProductQuantity: React.FC = () => {
+  const { item } = useItemContext()
+
+  return (
+    <span className="c-muted-1 t-body">
+      <FormattedMessage
+        id="store/product-list.quantity-units"
+        values={{ quantity: item.quantity }}
+      />
+    </span>
```

Comment:
Why not add a CSS handle to this component?
If there is no problem, that would be a good idea, since eventually, people will ask for it 😂

Reply:
I forgot to push the commit :joy: